web/src/stream_popover.ts

```diff
@@ -500,6 +504,90 @@ export async function build_move_topic_to_stream_popover(
             is_disabled;
     }
 
+    // Warn if any of current topic participants are NOT subscribed
+    // to the destination stream.
+    function warn_unsubscribed_participants(destination_stream_id: number): void {
+        $("#move_topic_modal .unsubscribed-participants-warning").remove();
+
+        const conversation_messages = message_util.get_messages_in_topic(
+            current_stream_id,
+            topic_name,
+        );
+
+        const active_human_participants = new ConversationParticipants(
+            conversation_messages,
+        ).visible();
+        const unsubscribed_participant_ids = [...active_human_participants].filter(
+            (user_id) => !stream_data.is_user_subscribed(destination_stream_id, user_id),
+        );
```

Comment:
@evykassirer can you advise @abdelrahman725 on how to fix this given #34259? I think we probably just want this function to `await` on having full subscriber data for the destination channel before trying to show the banner, and of course being careful to check that channel is still selected when that `await` finishes.

Reply:
Yup that sounds right. #34544 is a good example of adding this logic to a banner. Check that out and if you have further questions let me know!

Reply:
@evykassirer
So there is no difference between checking `stream_data.is_user_subscribed(..)` and checking if `user_id` is in `get_full_subscriber_set`, We just need to call and wait for `get_full_subscriber_set()` to ensure we get the full data, Am I understanding correctly?

Reply:
Yes that's the difference!